src/tyr/trace_serializer.cc

```diff
@@ -496,7 +496,7 @@ namespace tyr {
 std::string serializeTraceAttributes(
     const Api& request,
     const AttributesController& controller,
-    std::vector<std::tuple<float, float, std::vector<thor::MatchResult>>>& map_match_results) {
+    std::vector<std::tuple<float, float, std::vector<meili::MatchResult>>>& map_match_results) {
```

Comment:
See comments below about type alias. What are float values here?

Reply:
ill document the function that generates these. basically the floats are a normalized score, a raw score (cost in meters), and then the results of how each tracepoint was matched